models/Player.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+const userSchema = require('./User').schema;
+
+const playerSchema = new Schema({
+  name: String,
+	elo: Number,
+  team: String,
+  user: [userSchema]
```

Comment:
Why is this an array? Shouldn't each player account only be linked to a single user account? I can see user accounts having multiple player accounts, but not the other way around.